feg/gateway/services/s6a_proxy/servicers/test/test_s6a_server.go

```diff
@@ -109,37 +117,174 @@ func testHandleAIR(settings *sm.Settings) diam.HandlerFunc {
+	"\x73\x73\x2e\x65\x70\x63\x2e\x6d\x6e\x63\x30\x30\x31\x2e\x6d\x63" +
+	"\x63\x30\x30\x31\x2e\x33\x67\x70\x70\x6e\x65\x74\x77\x6f\x72\x6b" +
+	"\x2e\x6f\x72\x67\x00\x00\x01\x28\x40\x00\x00\x29\x65\x70\x63\x2e" +
+	"\x6d\x6e\x63\x30\x30\x31\x2e\x6d\x63\x63\x30\x30\x31\x2e\x33\x67" +
+	"\x70\x70\x6e\x65\x74\x77\x6f\x72\x6b\x2e\x6f\x72\x67\x00\x00\x00" +
+	"\x00\x00\x01\x0c\x40\x00\x00\x0c\x00\x00\x07\xd1\x00\x00\x05\x85" +
+	"\xc0\x00\x01\xc8\x00\x00\x28\xaf\x00\x00\x05\x86\xc0\x00\x00\x94" +
+	"\x00\x00\x28\xaf\x00\x00\x05\x8b\xc0\x00\x00\x10\x00\x00\x28\xaf" +
+	"\x00\x00\x00\x00\x00\x00\x05\xa7\xc0\x00\x00\x1c\x00\x00\x28\xaf" +
+	"\x15\x9a\xbf\x21\xca\xe2\xbf\x0a\xdb\xcb\xf1\x47\xef\x87\x74\x9d" +
+	"\x00\x00\x05\xa8\xc0\x00\x00\x14\x00\x00\x28\xaf\x66\x6a\x9b\x73" +
+	"\x6d\x41\x7a\x41\x00\x00\x05\xa9\xc0\x00\x00\x1c\x00\x00\x28\xaf" +
+	"\x6c\x3c\x22\x75\x44\x1a\x80\x00\x7e\x59\x79\x64\x47\x61\x43\x9a" +
+	"\x00\x00\x05\xaa\xc0\x00\x00\x2c\x00\x00\x28\xaf\x3b\x6d\xd0\x9b" +
+	"\x14\x6b\xfb\x53\x05\x1a\xa7\x5d\xe5\xe8\x93\xdb\x43\xd1\x00\xe4" +
+	"\x10\x48\x7d\x75\xcb\x26\x99\xe0\xe7\x80\xbb\x9e\x00\x00\x05\x86" +
+	"\xc0\x00\x00\x94\x00\x00\x28\xaf\x00\x00\x05\x8b\xc0\x00\x00\x10" +
+	"\x00\x00\x28\xaf\x00\x00\x00\x01\x00\x00\x05\xa7\xc0\x00\x00\x1c" +
+	"\x00\x00\x28\xaf\xd0\xb3\x82\xe2\xec\x53\xe3\xa6\xaf\xd8\x1c\xa4" +
+	"\x57\x92\xd8\xa6\x00\x00\x05\xa8\xc0\x00\x00\x14\x00\x00\x28\xaf" +
+	"\xf9\xf8\xcd\xcb\xc6\x50\x59\x47\x00\x00\x05\xa9\xc0\x00\x00\x1c" +
+	"\x00\x00\x28\xaf\x63\x82\xb8\x54\x48\x59\x80\x00\xf5\xaf\x37\xa5" +
+	"\xe9\x6d\x76\x58\x00\x00\x05\xaa\xc0\x00\x00\x2c\x00\x00\x28\xaf" +
+	"\x6d\xb8\x62\xd0\xd8\x54\x79\x51\x07\xc1\xbb\x97\xad\x14\x5d\xc0" +
+	"\x68\xd7\x6e\xa5\xbe\x1f\x7d\x80\x1a\x96\xe1\x48\xf5\x04\x89\x96" +
+	"\x00\x00\x05\x86\xc0\x00\x00\x94\x00\x00\x28\xaf\x00\x00\x05\x8b" +
+	"\xc0\x00\x00\x10\x00\x00\x28\xaf\x00\x00\x00\x02\x00\x00\x05\xa7" +
+	"\xc0\x00\x00\x1c\x00\x00\x28\xaf\x97\xce\x80\x32\x83\x47\x13\xdd" +
+	"\x31\xf7\x06\xb3\x77\xd6\x1a\x92\x00\x00\x05\xa8\xc0\x00\x00\x14" +
+	"\x00\x00\x28\xaf\x37\x2d\x89\xbc\x7c\x6b\x31\x9c\x00\x00\x05\xa9" +
+	"\xc0\x00\x00\x1c\x00\x00\x28\xaf\x8a\x9c\x9d\x2b\xdf\xf7\x80\x00" +
+	"\x77\xc5\xa4\x1a\xd3\xbc\x21\x29\x00\x00\x05\xaa\xc0\x00\x00\x2c" +
+	"\x00\x00\x28\xaf\x74\x60\x79\x2b\x8d\x5e\xb1\x62\xfd\x88\x28\xc2" +
+	"\x1a\x3b\xa0\xc5\x6e\x06\xed\xbf\x5b\x20\x54\x72\x50\x06\x36\xc5" +
+	"\xfa\xd9\x0b\x84\x00\x00\x02\x74\x80\x00\x00\x38\x00\x00\x28\xaf" +
+	"\x00\x00\x01\x0a\x40\x00\x00\x0c\x00\x00\x28\xaf\x00\x00\x02\x75" +
+	"\x80\x00\x00\x10\x00\x00\x28\xaf\x00\x00\x00\x01\x00\x00\x02\x76" +
+	"\x80\x00\x00\x10\x00\x00\x28\xaf\xbf\xff\xff\xff\x00\x00\x02\x74" +
+	"\x80\x00\x00\x38\x00\x00\x28\xaf\x00\x00\x01\x0a\x40\x00\x00\x0c" +
+	"\x00\x00\x28\xaf\x00\x00\x02\x75\x80\x00\x00\x10\x00\x00\x28\xaf" +
+	"\x00\x00\x00\x02\x00\x00\x02\x76\x80\x00\x00\x10\x00\x00\x28\xaf" +
+	"\x00\x01\x00\x00")
```

Comment:
btw, i have seen many tests using bytes. How do you normally generate and extract this bitstream?

Reply:
wireshark: copy as Escaped String, it requires less reformatting for Go.